sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/v2ResolutionPlans.scala

```diff
@@ -195,6 +196,10 @@ case class ResolvedFieldName(path: Seq[String], field: StructField) extends Fiel
   def name: Seq[String] = path :+ field.name
 }
 
+case object RootTableSchema extends FieldName {
```

Comment:
The reason to add it : https://github.com/apache/spark/pull/39942/files#r1102393297